src/lib/rnp.cpp

```diff
@@ -5415,8 +5410,9 @@ try {
     }
 
     /* encrypt secret key if requested */
-    if (op->password) {
-        prov = {.callback = rnp_password_provider_string, .userdata = (void *) op->password};
+    if (op->password.size()) {
```

Comment:
I think `!op->password.empty()` is more common

Reply:
Thanks, @dewyatt. Will fix and force-push it now.